apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -81,7 +82,7 @@ public function enqueue_assets() {
 		$screen = get_current_screen();
 
 		// Return early if we don't meet conditions to show templates.
-		if ( 'page' !== $screen->id || 'add' !== $screen->action ) {
+		if ( 'page' !== $screen->id  ) {
```

Comment:
This is a problem. By removing this the Modal appears on all screens. However, if we remove it then the JS for the meta isn't enqueued. Should we actually pass this `add` value into the JS and use that as a conditional to determine whether or not to show the Modal?